smarts/core/controllers/trajectory_tracking_controller.py

```diff
@@ -53,7 +54,13 @@ class TrajectoryTrackingController:
 
     @staticmethod
     def perform_trajectory_tracking_MPC(
-        trajectory, vehicle, state, dt_sec, prediction_horizon=5
+        trajectory: Tuple[
+            Sequence[float], Sequence[float], Sequence[float], Sequence[float]
+        ],
+        vehicle,
+        state,
```

Comment:
should the type be defined for these two?

Reply:
It is difficult to do at the moment since they do not have interfaces. We should definitely look into making a vehicle interface.